contracts/crowdfund/Crowdfund.sol

```diff
@@ -99,14 +102,22 @@ abstract contract Crowdfund is Implementation, ERC721Receiver, CrowdfundNFT {
     error OnlyPartyDaoError(address notDao);
     error OnlyPartyDaoOrHostError(address notDao);
     error OnlyWhenEmergencyActionsAllowedError();
+    error BelowMinimumContributionsError(uint96 contributions, uint96 minContributions);
+    error AboveMaximumContributionsError(uint96 contributions, uint96 maxContributions);
 
     event Burned(address contributor, uint256 ethUsed, uint256 ethOwed, uint256 votingPower);
     event Contributed(
+        address sender,
```

Comment:
is there any way we can keep the delegate in this event? would make it a lot easier for the front-end

Reply:
Will do 🫡